chainer/functions/connection/n_step_rnn.py

```diff
@@ -50,6 +50,19 @@ def _make_tensor_descriptor_array(xs):
     return PointerArray([d.value for d in descs], descs)
 
 
+def _get_cudnn_data_type(dtype):
```

Comment:
Should we redefine `cupy.cudnn.get_data_type` here?

Reply:
It would be enough, I will fix it!

Reply:
Fixed!